pyne/mesh.py

```diff
@@ -606,54 +674,61 @@ def __init__(self, mesh=None, structured=False,
             If mats is None, then no empty materials are created for the mesh.
 
             Unstructured mesh instantiation:
-                 - From iMesh instance by specifying: <mesh>
+                 - From PyMOAB core instance by specifying: <mesh>
                  - From mesh file by specifying: <mesh_file>
 
             Structured mesh instantiation:
-                - From iMesh instance with exactly 1 entity set (with BOX_DIMS
-                  tag) by specifying <mesh> and structured = True.
+                - From PyMOAB core instance with exactly 1 entity set
+                  (with BOX_DIMS tag) by specifying <mesh> and structured = True.
                 - From mesh file with exactly 1 entity set (with BOX_DIMS tag)
                   by specifying <mesh_file> and structured = True.
-                - From an imesh instance with multiple entity sets by
+                - From an PyMOAB instance with multiple entity sets by
                   specifying <mesh>, <structured_set>, structured=True.
                 - From coordinates by specifying <structured_coords>,
-                  structured=True, and optional preexisting iMesh instance
-                  <mesh>
+                  structured=True, and optional pre-existing PyMOAB core
+                  instance <mesh>
 
-            The "BOX_DIMS" tag on iMesh instances containing structured mesh is
-            a vector of floats it the following form:
+            The "BOX_DIMS" tag on PyMOAB core instances containing structured
+            mesh is a vector of floats it the following form:
```

Comment:
Something weird is happening, grammatically here.... maybe this is right:


```suggestion
            mesh is a vector of floats in the following form:
```